doc/develop/test/ztest.rst

```diff
@@ -595,10 +595,19 @@ twister with `--seed`
 
 Static configuration of ZTEST_SHUFFLE contains:
 
- - :kconfig:option:`CONFIG_ZTEST_SHUFFLE_SUITE_REPEAT_COUNT` - Number of iterations the test suite will run.
- - :kconfig:option:`CONFIG_ZTEST_SHUFFLE_TEST_REPEAT_COUNT` - Number of iterations the test will run.
+ - :kconfig:option:`CONFIG_ZTEST_SHUFFLE_SUITE_REPEAT_COUNT` - Number of iterations the test suite will run [DEPRECATED].
+ - :kconfig:option:`CONFIG_ZTEST_SHUFFLE_TEST_REPEAT_COUNT` - Number of iterations the test will run [DEPRECATED].
 
```

Comment:
```suggestion
```
I think it's better to remove those Kconfig options from the documentation right away to not advertise them anymore as they become deprecated and the replacing ones are documented right below.